<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1252</width>
    <height>682</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Diabetic Retinopathy Detection using machine Learning</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(181, 238, 255, 255), stop:1 rgba(255, 255, 255, 255))</string>
  </property>
  <widget class="QLabel" name="imgLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>621</width>
     <height>661</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>1069</x>
     <y>5</y>
     <width>171</width>
     <height>291</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Process</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="TrainButton">
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(103, 103, 103);
color: rgb(255, 255, 255);
border-radius:5px;
padding:5px;</string>
      </property>
      <property name="text">
       <string>Training</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="BrowseButton">
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(103, 103, 103);
color: rgb(255, 255, 255);
border-radius:5px;
padding:5px;</string>
      </property>
      <property name="text">
       <string>Browse Test Image</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="DetectRecogniseButton">
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(103, 103, 103);
color: rgb(255, 255, 255);
border-radius:5px;
padding:5px;</string>
      </property>
      <property name="text">
       <string> Recognize</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ExitButton">
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(103, 103, 103);
color: rgb(255, 255, 255);
border-radius:5px;
padding:5px;</string>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>310</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>370</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>14</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Prognosis &amp; Medication</string>
   </property>
  </widget>
  <widget class="QLabel" name="barimgLabel">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>10</y>
     <width>421</width>
     <height>351</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>400</y>
     <width>601</width>
     <height>271</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
